mipidsi/examples/parallel_ili9341_rp_pico/Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.1.0"
 
 [dependencies]
 cortex-m = "0.7.7"
-cortex-m-rt = "0.7.4"
+cortex-m-rt = "0.7.3"
```

Comment:
why the downgrade?

Reply:
`0.7.4` has been yanked from crates.io: https://crates.io/crates/cortex-m-rt/0.7.4